resources/qgis-resource-metadata.xsd

```diff
@@ -0,0 +1,133 @@
+            <xs:element name="constraints" type="rm:constraintsType" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="rights" type="xs:string" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="encoding" type="xs:string" minOccurs="0"/>
+            <xs:element name="crs" type="xs:string"/>
+            <xs:element name="extent" type="rm:extentType" maxOccurs="1"/>
+            <xs:element name="contact" type="rm:contactType" maxOccurs="unbounded"/>
+            <xs:element name="links" type="rm:linksType"/>
+        </xs:sequence>
+        <xs:attribute name="version" use="required" fixed="1.0"/>
+    </xs:complexType>
+    <xs:complexType name="constraintsType">
+      <xs:simpleContent>
+        <xs:extension base="xs:string">
+          <xs:attribute name="type" type="xs:string"/>
+        </xs:extension>
+      </xs:simpleContent>
+    </xs:complexType>  
+    <xs:complexType name="keywordsType">
+        <xs:sequence>
+            <xs:element name="keyword" maxOccurs="unbounded"/>
+        </xs:sequence>
+        <xs:attribute name="vocabulary" type="xs:string"/>
+    </xs:complexType>
+    <xs:complexType name="extentType">
+            <xs:sequence>
+                <xs:element name="spatial" type="rm:spatialType" maxOccurs="unbounded"/>
+                <xs:element name="temporal" type="rm:temporalType" minOccurs="0" maxOccurs="unbounded"/>
+            </xs:sequence>
+    </xs:complexType>
+    <xs:complexType name="spatialType">
+        <xs:attribute name="dimensions" type="xs:integer" default="2"/>
+        <xs:attribute name="crs" type="xs:string" use="required"/>
+        <xs:attribute name="minx" type="xs:decimal" use="required"/>
+        <xs:attribute name="miny" type="xs:decimal" use="required"/>
+        <xs:attribute name="minz" type="xs:decimal" use="optional"/>
+        <xs:attribute name="maxx" type="xs:decimal" use="required"/>
+        <xs:attribute name="maxy" type="xs:decimal" use="required"/>
+        <xs:attribute name="maxz" type="xs:decimal" use="optional"/>
+    </xs:complexType>
+    <xs:complexType name="temporalType">
+        <xs:choice>
+            <xs:element name="instant" type="xs:dateTime"/>
+            <xs:element name="period" type="rm:periodType"/>
+        </xs:choice>
+    </xs:complexType>
+    <xs:complexType name="periodType">
+        <xs:sequence>
+            <xs:element name="start" type="xs:dateTime"/>
+            <xs:element name="end" type="xs:dateTime" minOccurs="0"/>
+        </xs:sequence>
+        <xs:attribute name="indeterminatePosition" type="xs:boolean"/>
+    </xs:complexType>
+    <xs:complexType name="contactType">
+        <xs:sequence>
+            <xs:element name="name" type="xs:string"/>
+            <xs:element name="organization" type="xs:string" minOccurs="0"/>
+            <xs:element name="position" type="xs:string" minOccurs="0"/>
+            <xs:element name="address" type="rm:addressType" minOccurs="0"/>
```

Comment:
Should we allow multiple addresses for each contact? Eg postal address + street address?